### PR TITLE
fix(api): include more details in the money transfer schema

### PR DESCRIPTION
**Pull request by abuecker**

### CODE REVIEW

File: api/v2/examples/moneyTransfer.res.json [hunk 1]
```diff
@@ -1,11 +1,18 @@
 {
-  "transactionId":"3333",
-  "accountId": 1233454,
+  "transactionId": "59745728-1811-462d-b1dd-0a00f567791f",
+  "account": {
+    "id": "9889dc25-dec9-43b6-9165-4f8f42719287",
+    "name": "Individual Account",
+    "number": "847702886098"
```
Comment (kvhswong) [inline api/v2/examples/moneyTransfer.res.json]: Do you need the account number? The ID should be enough to interact with the API.

File: api/v2/examples/moneyTransfer.res.json [hunk 2]
```diff
@@ -1,11 +1,18 @@
 {
-  "transactionId":"3333",
-  "accountId": 1233454,
+  "transactionId": "59745728-1811-462d-b1dd-0a00f567791f",
+  "account": {
+    "id": "9889dc25-dec9-43b6-9165-4f8f42719287",
+    "name": "Individual Account",
```
Comment (kvhswong) [inline api/v2/examples/moneyTransfer.res.json]: Apologies for not digging up the account json earlier, but here are the available fields in an account, we should discuss if the names should be adjusted to match.

```
  "accounts": [
    {
      "id": "57c03455-0810-11e6-aa7b-12d4a7cc66a5",
      "description": "MALLOUK, PETER",
      "title": "PETER A MALLOUK ROLLOVER IRA",
      "accountType": "ROLLOVER IRA",
      "accountNumber": "936045619",
      "repCode": "ACFG",
      "dateOpened": "2015-02-27T00:00:00",
      "holdingsMarketDate": "2016-04-06",
      "accountValue": 3848.22,
      "cashEquivalent": 400.57,
      "origin": "ADVISOR",
      "forms": [],
      "bankInstructions": [],
```

File: api/v2/examples/moneyTransfer.res.json [hunk 3]
```diff
@@ -1,11 +1,18 @@
...
+    "number": "847702886098"
+  },
+  "bank": {
+    "name": "Grimes view City Bank",
+    "accountType": "CHECKING",
+    "number": "280633859366"
```
Comment (kvhswong) [inline api/v2/examples/moneyTransfer.res.json]: This is what is currently in the model for bank instructions. I can see if VEO provides more information.

```
    String bankInstructionId;
    String bankName;
    String accountName;
    String accountNumberLastFour;
    String permissionType;
```

File: api/v2/examples/moneyTransfer.res.json (hunk 1 above)
Comment (abuecker) [inline api/v2/examples/moneyTransfer.res.json]: The design displays the account number:

screenshot 2016-08-02 06 43 19

File: api/v2/examples/moneyTransfer.res.json (hunk 2 above)
Comment (abuecker) [inline api/v2/examples/moneyTransfer.res.json]: Let's expose both `description` and `title` so we have it if we need it.  I'll adjust the schema.

File: api/v2/examples/moneyTransfer.res.json (hunk 3 above)
Comment (abuecker) [inline api/v2/examples/moneyTransfer.res.json]: Let's work with what we have for now.  I'll adjust the schema.

**Comment by kvhswong** [comment]
Per my comment on HOME-517, there is an additional 2 bank properties, bankInstructionsId and permissionType. I can add these to the schema.

```
  bank: 
   { bankInstructionId: '5145555',
     bankName: 'BANK OF AMERICA, N.A. 1973',
     accountName: 'ARMANDA H GAZDICK',
     accountNumberLastFour: '1312',
     permissionType: 'ACHDepositsAndWithdrawals' } }
```

**Comment by kvhswong** [comment]
👍 , once this is merged I can merge my rest changes.

**Pull request merged**